.github/workflows/format.yml

```diff
@@ -0,0 +1,50 @@
+name: Fix
+
+on:
+  push:
+    branches-ignore:
+      - 'main'
+      - 'dev'
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'ci skip')"
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Cache ~/.pnpm-store
```

Comment:
Please rebase to the latest dev branch. We can use the cache option from `actions/checkout@v4`.

Reply:
done